content/learn/book/getting-started/ecs/_index.md

```diff
@@ -143,4 +143,4 @@ hello Zayna Nieves!
 
 Marvelous!
 
-**Quick Note**: "hello world!" might show up in a different order than it does above. This is because systems run in parallel by default whenever possible.
+**Quick note**: "hello world!" might show up in a different order than it does above. This is because systems run in parallel by default whenever possible.
```

Comment:
As a style thing, when we are talking about a **Specific Thing** (either a feature, a product, or a recurring concept), we capitalize the first letter of all words and bold them.

**Quick Note** is a specific, recurring thing (maybe not yet, but we will have **Quick Notes** in future book entries).